cmd/config.go

```diff
@@ -177,6 +181,29 @@ func LoadConfig() *Config {
 	config.StateChangeDir = viper.GetString("state-change-dir")
 	config.StateSyncerMempoolTxnSyncLimit = viper.GetUint64("state-syncer-mempool-txn-sync-limit")
 
+	// PoS Checkpoint Syncing
+	config.CheckpointSyncingProviders = viper.GetStringSlice("checkpoint-syncing-providers")
+	for _, provider := range config.CheckpointSyncingProviders {
+		if _, err := url.ParseRequestURI(provider); err != nil {
+			glog.Fatalf("Invalid checkpoint syncing provider URL: %v", provider)
+		}
+		// TODO: do we want to make a request to the checkpoint syncing provider to ensure it's valid?
+	}
+	// TODO: add default provider here based on network. However, if someone wants to sync w/o checkpoint
+	// syncing, they should be able to do so. How do we support this? another flag I guess.
+	if len(config.CheckpointSyncingProviders) == 0 && !config.Regtest {
```

Comment:
I think how you have it here is fine. The ideal way to do it would be to set a default flag value but we can't really do that because of testnet/mainnet needing different providers here.